doc/releases/migration-guide-4.2.rst

```diff
@@ -41,8 +41,8 @@ Boards
 * All boards based on a Nordic IC of the nRF54L series now default to not
   erasing any part of the internal storage when flashing. If you'd like to
   revert to the previous default of erasing the pages that will be written to by
-  the firmware to be flashed you can use the new ``--erase-pages`` command-line
-  switch when invoking ``west flash``.
+  the firmware to be flashed you can set the new ``--erase-mode`` command-line
+  switch when invoking ``west flash`` to ``pages``.
```

Comment:
The erase mode should be `ranges`, not `pages`. Also the commit message mentions `pages` as the possible mode, which is incorrect.

Reply:
Well spotted @anangl! Fixed.